pkg/network/kubemacpool.go

```diff
@@ -95,3 +109,26 @@ func generateRandomMacPrefix() ([]byte, error) {
 
 	return prefix, nil
 }
+
+func checkRange(startMac, endMac net.HardwareAddr) error {
+	for idx := 0; idx <= 5; idx++ {
+		if startMac[idx] < endMac[idx] {
+			return nil
+		}
+	}
+
+	return fmt.Errorf("invalid range start: %s end: %s", startMac.String(), endMac.String())
+}
+
+func checkCast(mac net.HardwareAddr) error {
+	bit := 1 & mac[0]
```

Comment:
please use binary:

```go
const multicastBit = 0b00000001
...
castBit := 0b00000001 & mac[0]
if castBit == multicastBit {
...
```

Reply:
I think GO does not support binary literals.
@phoracek

Reply:
you right!

Reply:
Anyway, please add inline comment describing what you do there. Something about the right-most bit from the first octet.